client/src/pages/admin/merge/MergeEmailAddresses.tsx

```diff
@@ -0,0 +1,180 @@
+import { Icon } from "@blueprintjs/core"
+import { IconNames } from "@blueprintjs/icons"
+import RemoveButton from "components/RemoveButton"
+import { ALIGN_OPTIONS, MERGE_SIDES, setAMergedField } from "mergeUtils"
+import React from "react"
+import { Button, Table } from "react-bootstrap"
+import utils from "utils"
+
+interface EmailAddressTableProps {
+  label: string
+  emailAddresses?: any[]
+  align?: (typeof ALIGN_OPTIONS)[keyof typeof ALIGN_OPTIONS]
+  mergeState?: any
+  dispatchMergeActions?: (...args: unknown[]) => unknown
+}
+
+const EmailAddressTable = ({
+  label,
+  emailAddresses,
+  align,
+  mergeState,
+  dispatchMergeActions
+}: EmailAddressTableProps) => {
+  const rows = emailAddresses ?? []
+  if (rows.length === 0) {
+    return <em>No {label.toLowerCase()} available</em>
+  }
+
+  const getMerged = () => mergeState?.merged?.emailAddresses ?? []
+
+  const upsertByNetwork = (list, item) => {
+    const idx = list.findIndex(e => e?.network === item?.network)
+    if (idx === -1) {
+      return [...list, item]
+    }
+    const next = [...list]
+    next[idx] = item
+    return next
+  }
+
+  const pickFromSide = item => {
+    if (!dispatchMergeActions) {
+      return
+    }
+    if (!mergeState?.merged && align && align !== "center") {
```

Comment:
This should use ALIGN_OPTIONS from mergeUtils.tsx, so `align !== ALIGN_OPTIONS.CENTER`.